spec/helpers/handle_helper_spec.rb

```diff
@@ -1,21 +1,20 @@
 require "spec_helper"
 include HandleHelper
 
-describe HandleHelper, unless: $in_travis do
+describe HandleHelper do
   let(:bills_file) { FactoryGirl.create(:bills_complete_file) }
-  `mysql -u "#{ENV["HANDLE_TEST_USERNAME"]}" < "#{Rails.root}"/spec/fixtures/files/handlesTEST.sql`
```

Comment:
Sorry, should've elaborated. HANDLE_TEST values and HANDLE (non-test) values are separated for a reason, and can't be used interchangeably (for the production reason I outlined above.

Unless I'm reading incorrectly, HANDLE_TEST was replaced by by the non test values in this spec file.

If we include travis, there are 8 handle databases

Dev - handles (fake), handles_test (fake)
Staging - handles (fake), handles_test (fake)
Travis - handles (fake), handles_test (fake)
Prod - handles (real!!), handles_test (fake)